docs/pages/guides/configuration-management/configuration-management.md

```diff
@@ -1,6 +1,6 @@
 ---
 layout: default
-title: Configuring your Cloud Native Infrastructure and Applications
```

Comment:
We should keep it here imo, as a user they would refer to apps as what we mean by designs.
But only remove mention of applications where it's mentioned/described as a meshery construct.